src/shared/components/palette/base-field/field-read.component.spec.ts

```diff
@@ -63,13 +73,60 @@ describe('FieldReadComponent', () => {
 
   let formGroup: FormGroup = new FormGroup({});
   let caseFields: CaseField[] = [CASE_FIELD];
+  let caseEditComponentStub: any;
+  const FORM_GROUP = new FormGroup({
+    'data': new FormGroup({'field1': new FormControl('SOME_VALUE')})
+  });
+  const wizardPage = createWizardPage([createCaseField('field1', 'field1Value')], false, 0);
+  const WIZARD = new Wizard([wizardPage]);
+  const caseField1 = new CaseField();
+  const firstPage = new WizardPage();
+  let cancelled: any;
+  const someObservable = {
+    'subscribe': () => new Draft()
+  };
+  const caseField2 = new CaseField();
+  let route: any;
+  const fieldTypeSanitiser = new FieldTypeSanitiser();
+  const formValueService = new FormValueService(fieldTypeSanitiser);
+  const formErrorService = new FormErrorService();
+  const caseFieldService = new CaseFieldService();
+  const pageValidationService = new PageValidationService(caseFieldService);
+  let dialog: any;
+  let caseEditPageComponent: CaseEditPageComponent;
 
   beforeEach(async(() => {
     paletteService = createSpyObj<PaletteService>('paletteService', [
       'getFieldComponentClass'
     ]);
     paletteService.getFieldComponentClass.and.returnValue(FieldTestComponent);
 
+    cancelled = createSpyObj('cancelled', ['emit']);
```

Comment:
@johnbenjamin-hmcts - Lines 104-129... as above

Reply:
@Daniel-Lam - We had to add CaseEditComponent reference to FieldReadComponent, so that it can be accessed by the child component(s). These lines of code were to declare and add a reference to CaseEditComponent in the spec file.